src/views/settings/Settings.vue

```diff
@@ -14,21 +14,32 @@ div
 
   hr
 
+  ReleaseNotificationSettings(v-if="isNewReleaseCheckEnabled")
+
+  hr(v-if="isNewReleaseCheckEnabled")
+
   CategorizationSettings
 
 </template>
 
 <script>
 import DaystartSettings from '~/views/settings/DaystartSettings.vue';
 import TimelineDurationSettings from '~/views/settings/TimelineDurationSettings.vue';
+import ReleaseNotificationSettings from '~/views/settings/ReleaseNotificationSettings.vue';
 import CategorizationSettings from '~/views/settings/CategorizationSettings.vue';
 
 export default {
   name: 'Settings',
   components: {
     DaystartSettings,
     TimelineDurationSettings,
+    ReleaseNotificationSettings,
     CategorizationSettings,
   },
+  data() {
+    return {
+      isNewReleaseCheckEnabled: !process.env.VUE_APP_ON_ANDROID,
```

Comment:
Neat, did not know you could pass arbitrary stuff to `vue-cli-service build` like that. 

Can't really seem to find any solid documentation for it (apart from a brief mention here: https://cli.vuejs.org/guide/mode-and-env.html which doesn't really talk about CLI parameters). I'd expect it to set `VUE_APP_ON` to `android`? What does the actual value here become?

Edit: Nevermind, found your edit to the webpack config below.